winit/src/clipboard.rs

```diff
@@ -51,4 +54,11 @@ impl iced_native::Clipboard for Clipboard {
     fn write(&mut self, contents: String) {
         self.write(contents)
     }
+
+    fn set_ime_position(&self, position: iced_core::Point) {
```

Comment:
I don't understand why is this method on clipboard? Shoudn't it be with window?

Reply:
> I don't understand why is this method on clipboard?

I thought it's natural that `TextInput` set IME window's position in [`on_event`](https://docs.rs/iced_native/0.4.0/iced_native/widget/trait.Widget.html#method.on_event) method and I chose to extend `Clipboard` struct to also handles IME to avoid larger diff. That's why this PR is WIP.

As I wrote above, I think there are 2 options.
1. Rename `Clipboard` to something and make also handles IME.
2. Pass another variable that handles IME to `on_event`.

> Shoudn't it be with window?

Is there a `window` that accessible from `Widget` trait? If so, maybe it's better to use that, but I also think that extend `Clipboard` is better still.